model/host/volume.go

```diff
@@ -0,0 +1,33 @@
+package host
+
+import (
+	"github.com/evergreen-ci/evergreen/db"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+type Volume struct {
+	ID        string `bson:"_id" json:"id"`
+	CreatedBy string `bson:"created_by" json:"created_by"`
+	Type      string `bson:"type" json:"type"`
+	Size      int    `bson:"size" json:"size"`
+}
+
+// Insert a volume into the volumes collection.
+func (v *Volume) Insert() error {
+	return db.Insert(VolumesCollection, v)
+}
+
+// Remove a volume from the volumes collection.
+func (v *Volume) Remove() error {
```

Comment:
Unless you ensure that no hosts are referencing this, I don't think you can remove the document entirely - it'll need to be a flag that it's been deleted in AWS

Reply:
It's called from https://github.com/evergreen-ci/evergreen/pull/2805/files/c3dfc87d54933a724d9955b592cc54946d84cac5#diff-e5707ed36e7443139f523d8951617a72R1149

Reply:
Oh touche. But [theoretically](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-deleting-volume.html) we can't delete a volume if it's attached to a host anyway, so I think this should be okay.